lib/spack/spack/test/cmd/commands.py

```diff
@@ -33,22 +30,10 @@ def test_subcommands():
     assert 'spack url parse' in out
     assert 'spack view symlink' in out
 
-    class Subcommands(ArgparseWriter):
-        def begin_command(self, prog):
-            assert prog in out
-
-    Subcommands().write(parser)
```

Comment:
To be honest with you, I have no idea what this test was supposed to do, and it no longer works, as expected, so I removed it.